model/project.go

```diff
@@ -1165,6 +1165,27 @@ func (p *Project) FindTaskGroup(name string) *TaskGroup {
 	return nil
 }
 
+// FindContainerFromProject finds the container configuration associated with a given task's Container field.
+func FindContainerFromProject(t task.Task) (*Container, error) {
+	v, err := VersionFindOneId(t.Version)
+	if err != nil {
+		return nil, errors.Wrapf(err, "finding version %s", t.Version)
```

Comment:
Need single quotes around the string. We do this specially for strings because they could potentially be empty, which makes the error message weird.